dashboard/src/components/Config/AppRepoList/AppRepo.scss

```diff
@@ -0,0 +1,12 @@
+.CertContainer {
+  margin: 0;
+  padding: 0;
+  margin-left: 0.2em;
+  background: inherit;
+}
+
+.CertContent {
+  // Copied from <code> style
+  font-family: "SFMono-Regular", Consolas, Liberation Mono, Menlo, Courier, monospace;
```

Comment:
We do not have any existing class in hex that could help us with that?

Reply:
So the hex style for this is to wrap code around a block `<pre><code>...`. The problem here is that `textarea` has its own style that overwrite that and the combination doesn't look good. That's why I need this.